### PR TITLE
Adds a new space ruin: Wrecked Cargoship

### PR DESCRIPTION
**Pull request by kano-dot**
## What Does This PR Do
Adds a space ruin with slightly spicy to questionable loot. Current loot is a subject to change.

## Why It's Good For The Game
As gateway content was removed exploring is in dire need of some attention. New content is good, bonus points if it has a story to tell.

## Images of changes
mostcurrent

## Testing
no issues with map files as far as i can tell though my word shouldn't be taken as i am completely new

## Changelog
:cl:
add: Added a new space ruin: wrecked cargoship
/:cl:

### CODE REVIEW

File: _maps/map_files/RandomRuins/SpaceRuins/wreckedcargoship.dmm [hunk 1]
```diff
@@ -0,0 +1,2004 @@
...
+	dir = 8;
+	pixel_x = -32
+	},
+/obj/effect/decal/cleanable/dirt,
+/turf/simulated/floor/pod/light,
+/area/ruin/space/wreck_cargoship)
+"ah" = (
+/obj/effect/decal/warning_stripes/yellow,
+/obj/machinery/door/poddoor/shutters{
+	id_tag = cargoshipblastdoor2
```
Comment (warriorstar-orion) [inline _maps/map_files/RandomRuins/SpaceRuins/wreckedcargoship.dmm:16]: contain id_tags in double quotes e.g. `id_tag = "cargoshipblastdoor2"`

**Comment by Aylong220** [comment]
1. Why not spawner?
image

2. That looks empty but secured, maybe add something?
image

3. Use spawners maaan - /obj/effect/spawner/window
image

4. Grilles under plastitanium spawners, that's bad
image

5. I think you can use here - /obj/effect/landmark/damageturf instead of icon_states
image

6. Flying bulb 
image
 
Everything else seems to be ok :gatto:

File: _maps/map_files/RandomRuins/SpaceRuins/wreckedcargoship.dmm (hunk 1 above)
Comment (kano-dot) [reply _maps/map_files/RandomRuins/SpaceRuins/wreckedcargoship.dmm:16]: oops

**Comment by kano-dot** [comment]
@Aylong220

1. well i just learned those exist, so that's useful

5. yeah i could've used landmarks there too but it's not the same and lacks some of the styles in the icon_state that i liked

Original plan was sticking with the scenario where a monster broke out, so i don't plan to add hostile creatures

**Comment by Sirryan2002** [comment]
Can we get a list with all the important/valuable loot? I just want to get an idea of what's available

**Comment by kano-dot** [comment]
> Can we get a list with all the important/valuable loot? I just want to get an idea of what's available

apologies for delayed response, current list of what's available in the ship:

- Double barreled shotgun
- 500 credits
- 15 ingots of gold

above three has placed inside a safe

- 3 abandoned crates
- An agent B tank
- 3 kitty ears
- Griefsky (the one with toy e-swords)
- two maintenance loot spawners (each spawns two item and the same one what's used in KC13)

**Comment by kano-dot** [comment]
i think i did something wrong while committing files, i didn't mean to request review, oups

**Review by S34NW — APPROVED** [review]

**Pull request merged**